### ISSUE TITLE
Umsetzung Service Provider Listing Logik (Filter, Sortierung)

### ISSUE THREAD

**Issue by NicoleNitschke** [neos/Neos.NeosIo#517]

**Comment by mhsdesign** [comment]
> The map is a beloved feature of the service provider list and should not be hidden under the long list. It should be placed right beside the service provider list instead. This way, the map and the list with its filters are visible above the fold.

Idk if left right really looks better ... (i cheated a bit and removed icons as well as made the cards smaller)

<img width="1727" alt="image" src="https://github.com/neos/Neos.NeosIo/assets/85400359/281e47b0-d354-4439-b388-645a366029dc">

Some examples if found:

<img width="1644" alt="image" src="https://github.com/neos/Neos.NeosIo/assets/85400359/9a6040d5-3920-4cdb-9762-ea5f92c4cd56">


<img width="1641" alt="image" src="https://github.com/neos/Neos.NeosIo/assets/85400359/8d90a86b-7a45-4532-a6c0-d430b172166b">

**Comment by NicoleNitschke** [comment]
@mhsdesign Yes, i don't think, that the map beside is a better solution, so I didn't change this layout yet. But I made first little changes in this PR: https://github.com/neos/Neos.NeosIo/pull/528
